### PR TITLE
Praise Will, Glorious Leader, My Mentor and Role Model, Emperor of the Galaxy

### PR DESCRIPTION
**Pull request by Guzaboo**
I love Will, my spirit animal is the water buffalo. I must go drink some coffee.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -68,6 +70,9 @@ client.on('ready', () => {
   console.log('Located IP channel: ' + ipChannel.name)
 
   checkIPChange(publicIP)
+  
+  let d = Math.round((new Date().getTime() - 1516492800000/*milliseconds since 1/1/1970 on 1/21/2018*/) / 86400000/*milliseconds in one day*/ + 1)
```
Comment (vpzomtrrfrt) [inline index.js]: this number is still unnecessarily magic

File: index.js [hunk 2]
```diff
@@ -101,13 +106,18 @@ client.on('message', message => {
...
     let simpleWord = messageSplit[i].replace(/[.,/#!$%^&*;:{}=\-_`~()]/g,'').replace(/\s{2,}/g,' ').toLowerCase().replace(/0|&#1086;/gi,'o').replace(/1/gi,'i').replace(' ', '')
-
+    let notSimpleWord = simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, '')
     console.log('Checking: ' + messageSplit[i])
     console.log('Simplified: ' + simpleWord)
-
-    console.log('Has profanity: ' + censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, '')))
-
-    if (censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, ''))) {
+
```
Comment (vpzomtrrfrt) [inline index.js]: why is 1 significant?

File: index.js [hunk 3]
```diff
@@ -101,13 +106,18 @@ client.on('message', message => {
...
-
-    console.log('Has profanity: ' + censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, '')))
-
-    if (censor.hasWord(simpleWord || simpleWord.replace(/[^\w\s]|(.)(?=\1)/gi, ''))) {
+    
+    let profane = (censor.hasWord(simpleWord || notSimpleWord) || englishWords.indexOf(simpleWord) != 1 && englishWords.indexOf(simpleWord) < d || englishWords.indexOf(notSimpleWord) != 1 && englishWords.indexOf(notSimpleWord) < d)
+    
+    console.log('Has profanity: ' + profane)
+    
+    let d = Math.round((new Date().getTime() - 1516492800000) / 86400000 + 1)
```
Comment (vpzomtrrfrt) [inline index.js]: code is still duplicated (and potentially unused?)

Comment (Guzaboo) [reply index.js]: heck you

Comment (vpzomtrrfrt) [reply index.js]: "You failed to fix the issue." - aragnon 2016

**Comment by LEARAX** [comment]
Annoying if reality doesn't match your fantasy, isn't it?

**Pull request closed without merging**